CHANGELOG.adoc

```diff
@@ -5,6 +5,64 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/spring-cloud/spring-cloud-gcp/commits/master[commit history] on GitHub.
 
+== 1.2.3.RELEASE (2020-05-29)
+
+=== General
+
+* Upgrade to latest libraries bom and sql socket factory (https://github.com/spring-cloud/spring-cloud-gcp/issues/2373[#2373])
+* Make transaction managers conditional on enabled flag for Spanner and Datastore (https://github.com/spring-cloud/spring-cloud-gcp/issues/2376[#2376])
+
+=== Logging
+
+NOTE: As we upgraded to the latest version of `google-cloud-logging-logback`, we picked up a https://github.com/googleapis/java-logging-logback/pull/43[breaking change] where the log entry payload is now written in JSON rather than plain text.
+So, if you're reading log entries back from Cloud Logging using `LogEntry.getPayload()`, make sure to cast the returned payload object to `JsonPayload` instead of `StringPayload`.
+
+=== Secret Manager 
+
+NOTE: This version introduced several breaking changes to Secret Manager property source.
+Please see the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.3.RELEASE/reference/html/#secret-manager-property-source)[reference documentation] for the new way for accessing secrets as properties.
+
+* Remove the version property in secret manager (https://github.com/spring-cloud/spring-cloud-gcp/issues/2270[#2270])
+* Secret manager template with project (https://github.com/spring-cloud/spring-cloud-gcp/issues/2283[#2283]) (https://github.com/spring-cloud/spring-cloud-gcp/issues/2284[#2284])
+* Create protocol for specifying secrets' project and versions (https://github.com/spring-cloud/spring-cloud-gcp/issues/2302[#2302])
+* Add secret manager autoconfigure property (https://github.com/spring-cloud/spring-cloud-gcp/issues/2363[#2363])
+
+=== Pub/Sub 
+
```

Comment:
```suggestion
* New async pull methods in `Pub/Sub Template`, as well as fully asynchronous `PubSubReactiveFactory.poll()` (https://github.com/spring-cloud/spring-cloud-gcp/pull/2227[#2227])
```

Reply:
I don't know how I missed this one. Good catch!